_docs/_latest/configure/index.md

```diff
@@ -44,6 +44,12 @@ Google Cloud Platform (GCP) resources.
 Configure Cloud Profiler to view and analyze CPU usage and memory-allocation of your Forseti application on a 
 Google Cloud Platform (GCP) interface.
 
+**[Configuring Forseti Visualizer]({% link _docs/latest/configure/forseti-visualizer/index.md %})**
```

Comment:
Link to the github page from here.  This way, we don't have a separate page like above that we have to maintain and keep in sync with the github description.  Less page is also more performant.

Reply:
Done